fuel-vm/src/interpreter/blockchain/code_tests.rs

```diff
@@ -80,20 +80,27 @@ fn test_load_contract() -> IoResult<(), Infallible> {
 fn test_code_copy() -> IoResult<(), Infallible> {
```

Comment:
Is there any coverage that needs to be added to the `blockchain` tests? Like we have all these for `ldc` [here](https://github.com/FuelLabs/fuel-vm/blob/master/fuel-vm/src/tests/blockchain.rs#L314). Specifically, it might be good to have a test that covers the case where contract size is larger than `$rD` and show the cost is proportional to the contract that is being copied rather than the bytes specified.